node/cli/src/command.rs

```diff
@@ -24,6 +24,8 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn ChainSpec>, String> {
 	Ok(match id {
 		"frequency_dev" | "dev" => Box::new(chain_spec::frequency_local::development_config()),
 		"frequency_local" => Box::new(chain_spec::frequency_local::local_testnet_config()),
+		"frequency_rococo" | "rococo" | "testnet" =>
+			Box::new(chain_spec::frequency_rococo::frequency_rococo_testnet()),
```

Comment:
here we either generate new specs or when we release specs we can add another case for frequency_rococo_pre that will just read the spec from file